README.md

```diff
@@ -587,6 +587,12 @@ Takes the given list of strings as a pipe and joins them on the provided string:
 {{$items | join ","}}
 ```
 
+#### `trim`
+Takes the provided input and trims all whitespace, tabs and newlines:
+```liquid
+{{ file "/etc/ec2_version"| trim }}
```

Comment:
I think we should rename this `trimSpace` and use Go's `strings.TrimSpace` function instead, since it's more in line with what is being asked.